uwblueprint/jump-math
# feat: student login

## Notion ticket link
Access Code page for Student Login (https://www.notion.so/uwblueprintexecs/Access-Code-page-for-Student-Login-efd8487245b54faca7bf7fe77410f235?pvs=4)

## Implementation description
* Created a new route for the Student Login page
* Used the AuthWrapper for the title, subtitle, and image
* Implemented pin input (frontend only)
* Added a back button

## Steps to test
1. Go to http://localhost:3000/student-login
2. Ensure everything looks correct according to Figma (https://www.figma.com/file/cRjJxm8087kkymfH4ieKau/Platform-Working-File?node-id=2154%3A81202&t=5kaMSKRfB4BWsKLN-4)
3. Ensure that the pin works as expected (the cursor will move to next input box with each numeric input)
4. Ensure that pin frontend looks correct when invalid (to do this, simply add "isInvalid" inside <PinInput> tag in StudentLogin.tsx)
5. Ensure that the back button brings the user back to the landing page

## What should reviewers focus on?
* that the pin behaves correctly
* the frontend matches the figma

## Checklist
